packages/react/src/auto/AutoTable.tsx

```diff
@@ -17,4 +17,5 @@ export type AutoTableProps<
   initialCursor?: string;
   initialDirection?: string;
   columns?: TableOptions["columns"];
+  onClickRow?: (row: Record<string, ColumnValueType>) => void;
```

Comment:
I don't love the 'onClickRow' name -- Let's go with onClick for now.

Reply:
Renamed.